src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -19,25 +21,24 @@ interface ContainerInfo {
 
 interface WorkloadPodLogsState {
   containerInfo?: ContainerInfo;
-  duration: string; // DurationInSeconds
+  duration: DurationInSeconds;
   loadingPodLogs: boolean;
   loadingPodLogsError?: string;
   podValue?: number;
   podLogs?: PodLogs;
+  tailLines: number;
 }
 
-const DurationDefault = '300';
-const DurationOptions = {
-  '60': 'Last 1m',
-  '300': 'Last 5m',
-  '600': 'Last 10m',
-  '1800': 'Last 30m',
-  '3600': 'Last 1h',
-  '10800': 'Last 3h',
-  '21600': 'Last 6h',
-  '43200': 'Last 12h',
-  '86400': 'Last 1d',
-  '604800': 'Last 7d'
+const TailLinesDefault = 500;
+const TailLinesOptions = {
+  '-1': 'All lines',
```

Comment:
"All lines" could be "By time"... And then the toolbar would be more human readable like "Tail: By time; Last X min"??
Well... I'm not sure about my suggestion; up to you ;)

Reply:
This is not really an option now that both filters active.

Reply:
Yes, with both active, not an option.